src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-form/pool-form.component.html

```diff
@@ -34,6 +34,12 @@
             <span class="invalid-feedback"
                   *ngIf="form.showError('name', formDir, 'uniqueName')"
                   i18n>The chosen Ceph pool name is already in use.</span>
+            <ng-container *ngIf="form.showError('name', formDir, 'rbdPool')">
+              <span class="invalid-feedback"
+                    i18n>It's not possible to create an RBD pool with '/' in the name.</span>
+              <span class="invalid-feedback"
```

Comment:
Do we really need to use two span's here? I know we're loosing the translation when we merge the two sentences into one.

Reply:
Done.